onnxruntime/core/framework/fallback_cpu_capability.cc

```diff
@@ -17,9 +17,16 @@ namespace {
 const int64_t kSmallInitializerThreshold = 100;
 
 bool IsSmallInitializer(const onnxruntime::GraphViewer& graph, const NodeArg* arg) {
-  const ONNX_NAMESPACE::TensorProto* initializer_tensor;
-  if (!graph.GetInitializedTensor(arg->Name(), initializer_tensor))
+  // 'true' in the function call is to let the searching for the initializer
+  // continue in the outer scopes of the current (sub-)graph if applicable
+  const ONNX_NAMESPACE::TensorProto* initializer_tensor =
+      graph.GetGraph().GetConstantInitializer(arg->Name(), true);
```

Comment:
GetConstantInitializer seems to be more strict as it also checks if the initializer cannot be overridden at runtime. Do we need this check?

Reply:
I had kept it this way because I thought it was "safer" to make decisions using constant initializers. But on further thought, all we check is the shape of the initializer to determine if it is "small" enough. I guess, we could do this for overridable initializers and not just constant initializers. I will update the check.

Reply:
Updated